src/exchange/hw-app-exchange/Exchange.ts

```diff
@@ -2,17 +2,18 @@ import Transport from "@ledgerhq/hw-transport";
 import { BigNumber } from "bignumber.js";
 import { TransportStatusError } from "@ledgerhq/errors";
 import invariant from "invariant";
-export const TRANSACTION_RATES = {
-  FIXED: 0x00,
-  FLOATING: 0x01,
-};
-export const TRANSACTION_TYPES = {
-  SWAP: 0x00,
-  SELL: 0x01,
-  FUND: 0x02,
-};
-type TransactionRate = typeof TRANSACTION_RATES[keyof typeof TRANSACTION_RATES];
-type TransactionType = typeof TRANSACTION_TYPES[keyof typeof TRANSACTION_TYPES];
+
+export const enum RateTypes {
+  FIXED = 0x00,
```

Comment:
enum property is usually defined by UpperCamerlCase

Reply:
Is const key necessary for enum in flow? It's not a case in TS.

Reply:
> enum property is usually defined in UpperCamelCase

Indeed, I'll update that

> Is const key necessary for enum in flow? It's not a case in TS.

Don't really understand what you mean by "necessary for enum in flow"?
We are not using flow anymore in LLC but TS.
Regarding the `const` keyword, it is not "necessary" per say but makes sense in this case since the enum is made up of const (i.e: non computed) members. cf. https://www.typescriptlang.org/docs/handbook/enums.html#const-enums